defender/wallet-migrator/approveTokens.js

```diff
@@ -0,0 +1,143 @@
+require('dotenv').config();
+const yaml = require('js-yaml');
+const fs = require('fs');
+const axios = require('axios');
+const { ethers } = require('ethers');
+const { RelayClient } = require('defender-relay-client');
+// grab secrets from .env file
+const GOERLI_RPC_URL = process.env.GOERLI_RPC_URL;
+const MAINNET_RPC_URL = process.env.MAINNET_RPC_URL;
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+// grab Defender API Key, Secret Key and Covalent API Key from .secrets/dev.yml file
+// we need to replace the "-" from yaml files, or it throws an error when importing
+let secretsFile = yaml.load(fs.readFileSync('defender/.secrets/dev.yml', 'utf8', { schema: 'JSON_SCHEMA' }));
```

Comment:
Because the user would already have to input their defender api and secret keys in the `/.secrets/dev.yml` file, I think it would be a smoother experience to just pull it from their so that they wouldn't have to input it again into a `.env` file